Makefile

```diff
@@ -1,8 +1,10 @@
+SHELL			:= /bin/bash
 # go options
 GO              ?= go
 LDFLAGS         :=
 GOFLAGS         :=
 BINDIR          := $(CURDIR)/bin
+ANTCTL_BINDIR   := $(CURDIR)/antctl-bin
```

Comment:
Should we just use bin dir (and for the container image put windows and MAC bins in different dirs)?
@antoninbas @tnqn : what are you opinions?

Reply:
I think the bin dir is good, not common to have separate bin dirs.

Reply:
OK, I would remove this dir. I think we don't need to care windows and macOS build since we would not build `antctl` in any image release.

Reply:
Actually I still like some way to publish the Windows and MAC versions, and thought packaging them into the image could be one way. We could also provide download from github on the same time.

Reply:
I want not to complicate your current PR further, so we could consider Windows and MAC builds later.

Reply:
Created #312 to track this issue.